appinfo/info.xml

```diff
@@ -1,4 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--
+  - SPDX-FileCopyrightText: 2013-2024 Nextcloud GmbH and Nextcloud contributors
```

Comment:
Should be changed to 
SPDX-FileCopyrightText: 2016-2024 Nextcloud GmbH and Nextcloud contributors
SPDX-FileCopyrightText: 2013 Bernhard Posselt <nukeawhale@gmail.com>